pkg/util/ansible_execution.go

```diff
@@ -119,32 +138,44 @@ func AnsibleExecution(
 			MountPath: "/runner/env/ssh_key",
```

Comment:
each key will need to be mounted at a unique path (based on the secret name). They will likely need to be mounted under the runner project dir, since /runner/env/ssh_key only has support for a single key (https://github.com/ansible/ansible-runner/issues/51). So something like `/runner/env/project/ssh_keys/<secret_name>`

Then when we generate the NodeSet inventory in inventory.go, I'd go ahead and set a group var there for ansible_ssh_private_key_file set to that same path.

Reply:
We are only passing one ssh key ansible runner though. It's specific to a node we are trying to reach. So this will have to be changed in deployment reconciliation first.

Reply:
you'll have to build a list of the ssh key secrets just like you build a list of the inventory secrets, and pass them in to the ansible execution.

ansible-runner won't use all the keys automatically, that's why I said you'll need to set a group var in the inventory with a path to the key. ansible-runner doesn't need to have any before hand knowledge of the path, it's just a variable set to the ssh key path.